src/js/port_handler.js

```diff
@@ -267,7 +267,7 @@ PortHandler.selectPort = function(ports) {
             const pathSelect = ports[i].path;
             const isWindows = (OS === 'Windows');
             const isTty = pathSelect.includes('tty');
-            const deviceRecognized = portName.includes('STM') || portName.includes('CP210');
+            const deviceRecognized = portName.includes('STM') || portName.includes('CP210') || portName.startsWith('SPR');
```

Comment:
I don't like too much to have exclusions for one vendor. I think SPR must include STM in the name. I suppose is a customized variant of STM right?

Reply:
The STM32H750 betaflight (legacy) target is shown as `/dev/tty/ACM0 - SPRacingH7EXTREME`

Reply:
@McGiverGim have removed it, agree this should not be vendor specific, as the board is auto selected anyway.